networking/hardware_networks/using-dpdk-and-rdma.adoc

```diff
@@ -5,15 +5,27 @@ include::modules/common-attributes.adoc[]
 
 toc::[]
 
-You can use Single Root I/O Virtualization (SR-IOV) network hardware with the Data Plane Development Kit (DPDK) and with remote direct memory access (RDMA).
 
-:FeatureName: The Data Plane Development Kit (DPDK)
-include::modules/technology-preview.adoc[leveloffset=+0]
+The containerized Data Plane Development Kit (DPDK) application is supported on {product-title}. You can use Single Root I/O Virtualization (SR-IOV) network hardware with the Data Plane Development Kit (DPDK) and with remote direct memory access (RDMA).
```

Comment:
Is the _DPDK application is supported_ sentence needed here? Isn't that implied by being in the docs. Seems like a release note rather than docs.

Reply:
I know you are not adding this assembly here. The entry in the topic map for this assembly is _Using DPDK and RDMA_. Is it worth updating the TOC to match the name of this assembly?

Reply:
I understand your suggestion, go ahead and change as you see fit